integ-test/src/test/java/org/opensearch/sql/calcite/big5/PPLBig5IT.java

```diff
@@ -49,257 +51,383 @@ public static void reset() throws IOException {
   }
 
   @Test
   public void term() throws IOException {
-    String ppl = sanitize(loadFromFile("big5/queries/term.ppl"));
+    String ppl = sanitize(loadExpectedQuery("term.ppl"));
     timing(summary, "term", ppl);
+    String expected = loadExpectedPlan("term.yaml");
+    assertYamlEqualsIgnoreId(expected, explainQueryYaml(ppl));
   }
 
   @Test
   public void terms_significant_1() throws IOException {
-    String ppl = sanitize(loadFromFile("big5/queries/terms_significant_1.ppl"));
+    String ppl = sanitize(loadExpectedQuery("terms_significant_1.ppl"));
     timing(summary, "terms_significant_1", ppl);
+    String expected = loadExpectedPlan("terms_significant_1.yaml");
+    assertYamlEqualsIgnoreId(expected, explainQueryYaml(ppl));
   }
 
   @Test
   public void terms_significant_2() throws IOException {
-    String ppl = sanitize(loadFromFile("big5/queries/terms_significant_2.ppl"));
+    String ppl = sanitize(loadExpectedQuery("terms_significant_2.ppl"));
     timing(summary, "terms_significant_2", ppl);
+    String expected = loadExpectedPlan("terms_significant_2.yaml");
+    assertYamlEqualsIgnoreId(expected, explainQueryYaml(ppl));
+  }
+
+  @Test
+  public void range_agg_1() throws IOException {
+    String ppl = sanitize(loadExpectedQuery("range_agg_1.ppl"));
+    timing(summary, "range_agg_1", ppl);
+    String expected = loadExpectedPlan("range_agg_1.yaml");
+    assertYamlEqualsIgnoreId(expected, explainQueryYaml(ppl));
+  }
+
+  @Test
+  public void range_agg_2() throws IOException {
+    String ppl = sanitize(loadExpectedQuery("range_agg_2.ppl"));
+    timing(summary, "range_agg_2", ppl);
+    String expected = loadExpectedPlan("range_agg_2.yaml");
+    assertYamlEqualsIgnoreId(expected, explainQueryYaml(ppl));
+  }
+
+  @Test
+  public void cardinality_agg_high() throws IOException {
+    String ppl = sanitize(loadExpectedQuery("cardinality_agg_high.ppl"));
+    timing(summary, "cardinality_agg_high", ppl);
+    String expected = loadExpectedPlan("cardinality_agg_high.yaml");
+    assertYamlEqualsIgnoreId(expected, explainQueryYaml(ppl));
+  }
+
+  @Test
+  public void cardinality_agg_high_2() throws IOException {
+    String ppl = sanitize(loadExpectedQuery("cardinality_agg_high_2.ppl"));
+    timing(summary, "cardinality_agg_high_2", ppl);
+    String expected = loadExpectedPlan("cardinality_agg_high_2.yaml");
+    assertYamlEqualsIgnoreId(expected, explainQueryYaml(ppl));
+  }
+
+  @Test
+  public void cardinality_agg_low() throws IOException {
+    String ppl = sanitize(loadExpectedQuery("cardinality_agg_low.ppl"));
+    timing(summary, "cardinality_agg_low", ppl);
+    String expected = loadExpectedPlan("cardinality_agg_low.yaml");
+    assertYamlEqualsIgnoreId(expected, explainQueryYaml(ppl));
+  }
+
+  protected String loadExpectedQuery(String fileName) throws IOException {
+    if (isCalciteEnabled()) {
+      try {
+        return loadFromFile("big5/queries/optimized/" + fileName);
```

Comment:
why have optimized folder? it required calcite enable?
IMO, PPL big5 query should be optimized queries.